manifests/r/RadioDJ/RadioDJ/2.0.4.3/RadioDJ.RadioDJ.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: RadioDJ.RadioDJ
+PackageVersion: 2.0.4.3
+PackageLocale: en-US
+Publisher: RadioDJ.ro

```

```suggestion
Publisher: Broadcast Mediaware, LLC
```
This must match control Panel